### PR TITLE
data: Kessi-ux

### PR DESCRIPTION
**Pull request by Kessi-ux**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Kessi-ux.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Uchenyi Kesandu",
+  "displayStatsPublic": true,
+  "type": "personal",
```
Comment (sital002) [inline data/Kessi-ux.json:4]:
```suggestion
```
```displayStatsPublic```  and ```type``` property isnot required

File: data/Kessi-ux.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "bio": "Backend Engineer",
+  "avatar": "https://github.com/Kessi-ux.png",
```
Comment (sital002) [inline data/Kessi-ux.json:6]:
```suggestion
```
```avatar``` property isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged.

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes to make the JSON pass and can now merge this PR.

**Pull request merged**